code/modules/surgery/surgery.dm

```diff
@@ -206,19 +246,3 @@
 				O.germ_level += germs
 
 	E.germ_level += germs
-
-/proc/sort_surgeries()
-	var/gap = GLOB.surgery_steps.len
-	var/swapped = 1
-	while(gap > 1 || swapped)
-		swapped = 0
-		if(gap > 1)
-			gap = round(gap / 1.247330950103979)
```

Comment:
How oddly specific 😂

Reply:
Somebody put a lot of effort into that number. And it was not even used in the previous code